sdk/tests/position.test.ts

```diff
@@ -0,0 +1,209 @@
+    erc20 = await Erc20.load(client, Network.Local, token1ContractHash)
+    await erc20.approve(ALICE, Key.Hash, invariantContractPackage, fee)
+
+    invariant = await Invariant.load(client, invariantContractHash, Network.Local)
+
+    feeTier = await callWasm(wasm.newFeeTier, { v: fee }, tickSpacing)
+    poolKey = await callWasm(wasm.newPoolKey, token0ContractPackage, token1ContractPackage, feeTier)
+
+    await invariant.addFeeTier(ALICE, feeTier)
+
+    await invariant.createPool(ALICE, poolKey, { v: 1000000000000000000000000n })
+
+    await invariant.createPosition(
+      ALICE,
+      poolKey,
+      lowerTickIndex,
+      upperTickIndex,
+      { v: 100000000000n },
+      { v: 1000000000000000000000000n },
+      { v: 1000000000000000000000000n }
+    )
+  })
+
+  it('create position', async () => {
+    const position = await invariant.getPosition(ALICE, 0n)
+
+    chai.assert.deepEqual(position.liquidity, { v: 100000000000n })
+    chai.assert.deepEqual(position.lowerTickIndex, lowerTickIndex)
+    chai.assert.deepEqual(position.upperTickIndex, upperTickIndex)
+    chai.assert.deepEqual(position.feeGrowthInsideX, { v: 0n })
+    chai.assert.deepEqual(position.feeGrowthInsideY, { v: 0n })
+    chai.assert.deepEqual(position.tokensOwedX, { v: 0n })
+    chai.assert.deepEqual(position.tokensOwedY, { v: 0n })
+  })
+
+  it('remove position', async () => {
+    await invariant.removePosition(ALICE, 0n)
+
+    assertThrowsAsync(invariant.getPosition(ALICE, 0n), wasm.InvariantError.PositionNotFound)
+    const positions = await invariant.getPositions(ALICE)
+    chai.expect(positions.length).to.equal(0)
+
+    assertThrowsAsync(invariant.getTick(poolKey, lowerTickIndex), wasm.InvariantError.TickNotFound)
+    assertThrowsAsync(invariant.getTick(poolKey, upperTickIndex), wasm.InvariantError.TickNotFound)
+
+    const isLowerTickInitialized = await invariant.isTickInitialized(poolKey, lowerTickIndex)
+    chai.expect(isLowerTickInitialized).to.equal(false)
+
+    const isUpperTickInitialized = await invariant.isTickInitialized(poolKey, upperTickIndex)
+    chai.expect(isUpperTickInitialized).to.equal(false)
+  })
+
+  it('transfer position', async () => {
+    await invariant.transferPosition(ALICE, 0n, Key.Account, bobAddress)
+
+    assertThrowsAsync(invariant.getPosition(ALICE, 0n), wasm.InvariantError.PositionNotFound)
+    const position = await invariant.getPosition(BOB, 0n)
+
+    chai.assert.deepEqual(position.liquidity, { v: 100000000000n })
+    chai.assert.deepEqual(position.lowerTickIndex, lowerTickIndex)
+    chai.assert.deepEqual(position.upperTickIndex, upperTickIndex)
+    chai.assert.deepEqual(position.feeGrowthInsideX, { v: 0n })
+    chai.assert.deepEqual(position.feeGrowthInsideY, { v: 0n })
+    chai.assert.deepEqual(position.tokensOwedX, { v: 0n })
+    chai.assert.deepEqual(position.tokensOwedY, { v: 0n })
+  })
+
+  it('claim fee', async () => {
+    const [tokenX, tokenY] = (await callWasm(
```

Comment:
Add function to hide the call to `callWasm`. For example: https://github.com/invariant-labs/protocol-cspr/pull/109/files#diff-a9bac152ce35c823f8ae4b7447382f03ab3437980a44b2806209d5b847e9d66bR304

Description here: https://github.com/invariant-labs/protocol-cspr/pull/109#discussion_r1521834191